engine/gamesys/src/gamesys/components/comp_model.cpp

```diff
@@ -370,6 +372,7 @@ namespace dmGameSystem
         component->m_World = Matrix4::identity();
         component->m_DoRender = 0;
         component->m_FunctionRef = 0;
+        component->m_RenderConstants = 0;
```

Comment:
Since the component struct is using POD semantics, the constructor of the render constants was never called.

It was wrong to use the renderconstants the way we did (I ported most of them, but left a few "for later")